pkg/controller/webhook/util.go

```diff
@@ -58,7 +56,7 @@ func NewValidatingResource(resourcePluralName string) schema.GroupVersionResourc
 
 // Allowed returns true if the admission request for the plural name of the
 // resource passed in should be allowed to pass through, false otherwise.
-func Allowed(a *admissionv1beta1.AdmissionRequest, pluralResourceName string, status *admissionv1beta1.AdmissionResponse) bool {
+func Allowed(a admission.Request, pluralResourceName string, status *admission.Response) bool {
```

Comment:
I know this is old code but why don't these methods return an `admission.Response` instead of mutating one that is passed in?

Reply:
I'm happy to have this tackled in a future PR.

Reply:
tracking it in https://github.com/kubernetes-sigs/kubefed/issues/1267